cmd/content-mirror/template_nginx.go

```diff
@@ -75,6 +75,34 @@ http {
       proxy_ssl_certificate_key {{ .KeyPath }};
       {{- end }}
       {{- end }}
+
+		# Do not cache repomd.xml. These need to be pulled from the
+		# mirrored server each time. When a yum repository is rebuilt, references in an old
+		# copy of repomd.xml will no longer resolve - resulting in 404s.
+		location ~ ^.*/(repodata/repomd\.xml) {
```

Comment:
Can we shorten the caching duration instead to be 60s or something?

Reply:
Looking into it..